ci/pipeline.yaml

```diff
@@ -598,4 +915,565 @@ jobs:
                 fi
 
                 echo "**** Public release, create github.com release $VERSION. ****"
-                ./ci/create-github-release.sh $OLM_RELEASE_VERSION $GH_API_TOKEN
+                ./ci/scripts/create-github-release.sh $OLM_RELEASE_VERSION $GH_API_TOKEN
+        on_success:
+          put: gh-status
+          inputs: [ pipeline-source ]
+          params: { state: success, context: operator-olm-github-release/build }
+        on_failure:
+          put: gh-status
+          inputs: [ pipeline-source ]
+          params: { state: failure, context: operator-olm-github-release/build }
+        on_error:
+          put: gh-status
+          inputs: [ pipeline-source ]
+          params: { state: error, context: operator-olm-github-release/build }
+        on_abort:
+          put: gh-status
+          inputs: [ pipeline-source ]
+          params: { state: error, context: operator-olm-github-release/build }
+
+  - name: olm-release-pr-creation
+    max_in_flight: 1
+    plan:
+      - get: instana-agent-operator-release
+        version: every #fetch the latest operator release every time the job is triggered
+        params:
+          globs:
+            - olm-*.zip
+      - get: agent-operator-release-source
+        trigger: true
+        passed: [operator-olm-github-release]
```

Comment:
Will this potentially trigger the PR creation twice per release?
So the first time when the `./ci/scripts/create-github-release.sh $OLM_RELEASE_VERSION $GH_API_TOKEN` is called in `operator-olm-github-release` which should lead to an update on the concourse resource and the other once there was a file change?
Not sure if that is an issue or not

Reply:
`instana-agent-operator-release` no longer triggers the job actually, now it's only triggered with `agent-operator-release-source` after `operator-olm-github-release` is successful. `version: every` just ensures that we fetch the latest version of the release when the job is triggered

Reply:
ah, okay, I see, makes sense, thanks :slightly_smiling_face: